src/git/gitUri.ts

```diff
@@ -365,7 +366,7 @@ export class GitUri extends ((Uri as any) as UriEx) {
 
         if (normalizedFileName.startsWith(normalizedRepoPath)) return normalizedFileName;
 
-        return Strings.normalizePath(paths.join(normalizedRepoPath, normalizedFileName));
+        return Strings.normalizePath(url.resolve(repoPath, fileName));
```

Comment:
Need to ensure that the paths all have the same `/` slashes

```suggestion
        return Strings.normalizePath(url.resolve(normalizedRepoPath, normalizedFileName));
```